packages/get-credentials/test/get-credentials.spec.js

```diff
@@ -0,0 +1,104 @@
+import fs from 'fs'
+import sinon from 'sinon'
+
+import {
+  getCredentials,
+  getCredentialsFromEnvironment,
+  setCredentialsFromEnvFile,
+} from '../src/index'
+import { homepage } from '../package.json'
+
+describe('getCredentials', () => {
+  test('should reject error when missing project key argument', (done) => {
+    getCredentials()
+      .then(done.fail)
+      .catch((errors) => {
+        expect(errors[0].message).toBe('Project Key is needed')
+        done()
+      })
+  })
+
+  test('should reject an array of all errors', (done) => {
+    getCredentials('forgery')
+      .then(done.fail)
+      .catch((errors) => {
+        expect(errors.length).toBe(2)
+        expect(errors[0].message).toBeTruthy()
+        expect(errors[1].message).toBeTruthy()
```

Comment:
Since both the `dotenv` fails and the retrieval from the environment variable fails. I've changed the assertions to be more descriptive.